templates/options.conf.erb

```diff
@@ -26,7 +26,7 @@ listen-on-v6 { <%= scope.lookupvar('::dns::listen_on_v6') %>; };
 allow-recursion { <%= scope.lookupvar('::dns::allow_recursion').join("; ") %>; };
 <% end -%>
 
-<% if (@osfamily =~ /^(FreeBSD|DragonFly)$/) -%>
+<% if (@os['family'] =~ /^(FreeBSD|DragonFly)$/) -%>
```

Comment:
I'd prefer to use the facts hash rather than relying on top level variables.
```suggestion
<% if (@facts['os']['family'] =~ /^(FreeBSD|DragonFly)$/) -%>
```